node/subsystem/src/messages.rs

```diff
@@ -237,23 +237,44 @@ pub enum NetworkBridgeMessage {
 	/// Second parameter, tells what to do if we are not yet connected to the peer.
 	SendRequests(Vec<Requests>, IfDisconnected),
 
-	/// Connect to peers who represent the given `validator_ids`.
+	/// Connect to peers.
 	///
-	/// Also ask the network to stay connected to these peers at least
-	/// until the request is revoked.
-	/// This can be done by dropping the receiver.
-	ConnectToValidators {
-		/// Ids of the validators to connect to.
-		validator_ids: Vec<AuthorityDiscoveryId>,
-		/// The underlying protocol to use for this request.
+	/// Also ask the network to stay connected to these peers at least until the request is
+	/// revoked.  This can be done by dropping the corresponding `requests` sender.
```

Comment:
This paragraph is outdated now, how do we drop a specific `ConnectMessage` request (see my comments below)?
These docs should probably go to `ConnectMessage`.